packages/stats/gbstats/gbstats.py

```diff
@@ -25,11 +25,41 @@
     SampleMeanStatistic,
     RatioStatistic,
     RegressionAdjustedStatistic,
-    Statistic,
+    TestStatistic,
 )
 from gbstats.messages import raise_error_if_bayesian_ra
 
 
+@dataclass
+class AnalysisSettingsForStatsEngine:
+    var_names: List[str]
+    weights: List[float]
+    baseline_index: int
+    dimension: str
+    stats_engine: str
+    sequential_testing_enabled: bool
+    sequential_tuning_parameter: float
+    difference_type: str
+    phase_length_days: float
+    alpha: float
+    max_dimensions: int
+
+
+@dataclass
+class MetricDataForStatsEngine:
+    metric: str
+    rows: List[Dict[str, Union[str, int, float]]]
+    inverse: bool
+    multiple_exposures: int
+
+
+@dataclass
+class DataForStatsEngine:
+    var_id_map: Dict[str, int]
+    metrics: List[MetricDataForStatsEngine]
+    analyses: List[AnalysisSettingsForStatsEngine]
```

Comment:
Set up to mirror the typescript interfaces, build below in `process_experiment_results`.